README.md

```diff
@@ -66,3 +66,111 @@ that can be used to quickly bring up a GKE cluster and configure
 `kubectl` and `helm` locally. This can be used to quickly spin up a test
 cluster for acceptance tests. Unit tests _do not_ require a running Kubernetes
 cluster.
+
+### Writing Unit Tests
+
+Changes to the Helm chart should be accompanied by appropriate unit tests.
+
+#### Formatting
+
+- Put tests in the test file in the same order as the variables appear in the `values.yaml`. 
+- Start tests for a chart value with a header that says what is being tested, like this:
+    ```
+    #--------------------------------------------------------------------
+    # annotations
+    ```
+
+- Name the test based on what it's testing in the following format:
+    ```
+    { section being tested }: { short description of the test case }
```

Comment:
This is fine but it might be even more clear if this line included the actual syntax you need to name a test (more like an example) e.g.

```
@test "<section being tested>: <short description of test case>" {
```

Or alternatively an example is given right after to make it completely obvious that in the examples below that string is what was being referred to here.

Reply:
Completely reasonable. I've updated the formatting to include the full line and made a note to that effect.